astropy/io/votable/tests/test_vo.py

```diff
@@ -728,9 +728,7 @@ def table_from_scratch():
 @np.errstate(over="ignore")
 def test_open_files():
     for filename in get_pkg_data_filenames("data", pattern="*.xml"):
-        if filename.endswith("custom_datatype.xml") or filename.endswith(
-            "timesys_errors.xml"
-        ):
+        if filename.endswith(("custom_datatype.xml", "timesys_errors.xml")):
             continue
         parse(filename)
```

Comment:
The proposed change does address Ruff rule PIE810, but this code deserves to be refactored more aggressively:
```suggestion
        if not filename.endswith(("custom_datatype.xml", "timesys_errors.xml")):
            parse(filename)
```

Reply:
Thanks for the suggestion!